src/config/Config.zig

```diff
@@ -4615,6 +4615,13 @@ pub const Keybinds = struct {
                 .{ .jump_to_prompt = 1 },
             );
 
+            // Toggle command palette, matches VSCode
+            try self.set.put(
```

Comment:
We can move this into the cross-platform keybinds now and use our `ctrlOrSuper` helper since the bindings match.